includes/definitions/quanta.yaml

```diff
@@ -6,9 +6,10 @@ over:
     - { graph: device_bits, text: 'Device Traffic' }
 discovery:
     - sysObjectId:
-        - .1.3.6.1.4.1.4413
-        - .1.3.6.1.4.1.7244
-    - sysDescr:
-        - vxworks
-        - quanta
-        - LB9, Runtime Code
+          - .1.3.6.1.4.1.7244
+    -
+      sysObjectId:
+          - .1.3.6.1.4.1.4413
+      sysDescr_regex:
+          - '/vxworks/i'
```

Comment:
I know you are :) But, we don't know their devices don't have varying case sensitivity. We've not yet come across devices being detected as Quanta.